polygeist/tools/cgeist/Lib/clang-mlir.cc

```diff
@@ -1382,12 +1400,12 @@ ValueCategory MLIRScanner::CommonFieldLookup(clang::QualType CT,
           ValueCategory(commonGEP, /*isReference*/ true).getValue(Builder);
     return ValueCategory(commonGEP, /*isReference*/ true);
   }
-  auto MT = val.getType().cast<MemRefType>();
-  auto shape = std::vector<int64_t>(MT.getShape());
-  if (shape.size() > 1) {
-    shape.erase(shape.begin());
+  auto MT = Val.getType().cast<MemRefType>();
+  auto Shape = std::vector<int64_t>(MT.getShape());
```

Comment:
Can we use `llvm::SmallVector` here instead?

Reply:
done.